src/app/code/community/Mageone/Qps/controllers/Adminhtml/QpsController.php

```diff
@@ -12,7 +12,7 @@ public function indexAction(): void
 
     public function editAction()
     {
-        $id    = $this->getRequest()->getParam('id');
+        $id = $this->getRequest()->getParam('id');
```

Comment:
be careful with these whitespace-only changes. normally no problem, but if customers want to review the changes, it is harder.

Reply:
This kind of white space shouldn't even happen. It’s just bad. All unnecessary white space is.

Reply:
I'm happy to challenge that opinion. With a beer (or something else) on a nice evening :-)